src/Microsoft.OpenApi/Services/OpenApiServerComparer.cs

```diff
@@ -54,8 +54,6 @@ public class OpenApiServerComparer : OpenApiComparerBase<OpenApiServer>
                 () => comparisonContext
                     .GetComparer<IDictionary<string, OpenApiServerVariable>>()
                     .Compare(sourceServer.Variables, sourceServer.Variables, comparisonContext));
-
-            // To Do compare extensions
```

Comment:
Why are we removing these "// To Do compare extensions" comments?

Reply:
i will be tracking this work as an issue on github instead of adding these ToDo

---
In reply to: [225337695](https://github.com/Microsoft/OpenAPI.NET/pull/331#discussion_r225337695) [](ancestors = 225337695)